components/automate-chef-io/content/docs/eas.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Chef EAS"
```

Comment:
i kinda want this to be `Enterprise Application Stack` in the heading and left nav.. or with Chef if it fits... idk acronyms are difficult

Reply:
I suspect that people are slinging around "EAS" without much explanation, so I want the acronym to come up #1 in search. Putting it in the title and the nav shoud do the trick. I can't tweak the search results in Automate.

Reply:
Ah! That makes total sense, thanks for explaining the why.